lib/js/components/Pagination/Pagination.vue

```diff
@@ -333,8 +366,23 @@ export default {
 
 			return navigationItems;
 		},
+		navigationItemsForDropdown() {
+			if (this.lastPage <= FIRST_PAGE_NUMBER) {
+				return [];
+			}
+
+			return this.getRange(1, this.lastPage).map((item) => ({
+				label: `${item} (${(item - 1) * this.itemsPerPage + 1} - ${
+					this.itemsPerPage * item
```

Comment:
What if the number of items isn't equal to the multiple of items per page? E.g. 30 items per page and there are 93 items. For the last page it would show 91-120, while in reality there is 91-93.